### DISCUSSION TITLE
加州通過人工智慧安全法案 加強監管AI

### DISCUSSION THREAD

**Discussion by Raiden1121** [1101-GS4719-C/Python-Community-GS4719-C#2093] [General]
https://www.cna.com.tw/news/ait/202408290084.aspx?topic=3437
隨著AI的發展，關於管制AI的法條也越來越重要。
這次加州提出的法條是關於要求大型「前沿」AI模型的開發者採取預防措施，例如部署前測試、模擬駭客攻擊、安裝網路安全防護措施以及為吹哨者提供保護。不過依然有人反對這項法案，認為這會影響AI的發展。
我比較贊同成立此法條，因為我認為即使AI發展再強大，沒有法律的管制，最終仍可能脫離正軌。

**Comment by Chattenoire** [comment]
關於AI的整體思考，我推薦Nick Bostrom所寫的超智慧(Superintelligence)。作者自己本身不僅理工人文素養皆有，且學術影響力可說是近50年來在社科領域最大。此外，更難得的是，這本書2014年就寫好了；恰恰印證了哲學家的vision總是能夠超越時代。

簡單的理解可以看English wikipedia如下:
https://en.wikipedia.org/wiki/Superintelligence:_Paths,_Dangers,_Strategies

**Comment by 112121py** [comment]
我也認同這項法條的成立，雖然如果對科技有明顯的限制會導致沒有辦法非常快速的前進，可是也能讓這項技術在進步的同時，讓大家意識到「為可能帶來的負面影響做準備」這件事情是非常重要的，不然當它發展到超過我們人類可以控制的範圍時，無論是有心人士的利用，或甚至視這項技術本上，都有可能會為我們帶來很嚴重的過度快速發展的反噬

**Comment by yuan3x2** [comment]
隨著科技發展，我認同法條確實應該與時俱進，以因應風險，降低不可控性，也能夠對有心人士進行一定程度的防範，此外，AI發展本身就存在著許多風險和倫理問題，因此更應該訂定相關法規進行規範。
